kubernetes/data_source_kubernetes_pod.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 func dataSourceKubernetesPod() *schema.Resource {
-	podSpecFields := podSpecFields(false, false, false)
+	podSpecFields := podSpecFields(false, false)
```

Comment:
I removed the `isDeprecated` field from the podSpecFields function because we're about to release a major update. The replication controller resource was the one using isDeprecrated, since originally the provider was created with a replication controller spec that didn't match the API. The issue was fixed in January 2019 (commit d5ff12c5b137b7a04be60e640d3c7c8b654d659e) and the affected resources have had deprecation warnings since then.